v6: Can't resolve 'react/jsx-runtime' in '/usr/src/app/node_modules/nuka-carousel/dist'

# Bugs and Questions

### Prerequisites

<!-- Feel free to delete this section if you have checked off all of the following. -->

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using? v6.0.1
- What version of React are you using? v17.0.2
- What browser are you using? N/A

### Describe the Problem

Using create-react-app and installing the latest version of nuka-carousel causes build step to fail with a compilation error.
```
#16 2.213 Creating an optimized production build...
#16 39.49 Failed to compile.
#16 39.49 
#16 39.49 Module not found: Error: Can't resolve 'react/jsx-runtime' in '/usr/src/app/node_modules/nuka-carousel/dist'
#16 39.49 Did you mean 'jsx-runtime.js'?
#16 39.49 BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
#16 39.49 (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
#16 39.49 The extension in the request is mandatory for it to be fully specified.
#16 39.49 Add the extension to the request.
#16 39.49 
#16 39.49 
#16 39.59 error Command failed with exit code 1.
#16 39.59 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
#16 ERROR: executor failed running [/bin/sh -c REACT_APP_ENV=$REACT_APP_ENV yarn build]: exit code: 
```

<!--
  It's easier to show us than tell us what's going wrong with your code.
  Because of this, we ask that you choose one of the following methods to help us reproduce the bug:

  A. Create a public minimal repository that we can `git clone`, with install + error reproduction steps in the README.
  B. Create a sandbox (example: codesandbox.io/s/n39x25), reproduce your issue in the code, and paste the link here.
  C. Open up a PR, include "WIP" and the Issue # in the title, and point us to the failing regression tests.
-->

#### Expected behavior:
Working the same way as v5

#### Actual behavior
Fail to compile on v6 in create-react-app environment

### Additional Information

--- a/.changeset/brave-melons-rhyme.md
+++ b/.changeset/brave-melons-rhyme.md
@@ -1,0 +1,5 @@
+---
+'nuka-carousel': patch
+---
+
+Fix peer dependency version for React in package.

--- a/docs/installation.mdx
+++ b/docs/installation.mdx
@@ -33,6 +33,12 @@ pnpm add nuka-carousel
   </TabItem>
 </Tabs>
 
+:::info
+
+Nuka Carousel has a peer dependency on React 18.
+
+:::
+
 ### Basic Usage
 
 ```jsx

--- a/packages/nuka/package.json
+++ b/packages/nuka/package.json
@@ -66,8 +66,8 @@
     "webpack-dev-server": "^4.6.0"
   },
   "peerDependencies": {
-    "react": "^16.0.0 || ^17.0.1 || ^18.0.0",
-    "react-dom": "^16.0.0 || ^17.0.1 || ^18.0.0"
+    "react": ">=18.0.0",
+    "react-dom": ">=18.0.0"
   },
   "resolutions": {
     "jsdom": "^14.0.0"

